packages/mobile/src/fees/saga.ts

```diff
@@ -27,6 +28,16 @@ const placeholderSendTx: BasicTokenTransfer = {
 
 export function* estimateFeeSaga({ feeType }: EstimateFeeAction) {
   Logger.debug(`${TAG}/estimateFeeSaga`, `updating for ${feeType}`)
+
+  const balance = yield select(stableTokenBalanceSelector)
+  if (balance === '0') {
```

Comment:
Might be good to parse the balance and check it here, in case it's null or empty string

Reply:
What should happen if the balance it's null or empty?

Reply:
The same thing :)